### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by skhomuti**
human-readable https://github.com/lidofinance/linters/blob/readme/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,67 @@
...
-Validate:
-
-- Python
-- Go
-- JavaScript
-- TypeSctipt
-- Dockerfile
-- Gitleaks
\ No newline at end of file
+This is GitHub actions and workflows with linters
```
Comment (dgusakov) [inline README.md]:
```suggestion
This is the place where GitHub actions and workflows with linters live
```

File: README.md [hunk 2]
```diff
@@ -1,10 +1,67 @@
...
+- [Pylint](http://pylint.pycqa.org) for code style and errors
+- [Black](http://black.readthedocs.io) for autoformatting
+- [ISort](https://github.com/PyCQA/isort) for imports ordering
+- [Bandit](http://bandit.readthedocs.io) for common python security issues (via semgrep)
+
+You can use it as a workflow or an action, as you wish.
+```yaml
+jobs:
+  # As a workflow
+  security:
```
Comment (dgusakov) [inline README.md]:
```suggestion
  python-lint-workflow:
```

File: README.md [hunk 3]
```diff
@@ -1,10 +1,67 @@
...
+    uses: lidofinance/linters/.github/workflows/python.yml@master
+  
+  # Or as an action
+  security-action:
```
Comment (dgusakov) [inline README.md]:
```suggestion
  python-lint-action:
```

File: README.md [hunk 4]
```diff
@@ -1,10 +1,67 @@
...
+    name: Check security issues
```
Comment (dgusakov) [inline README.md]:
```suggestion
    name: Lint python code
```

**Review by dgusakov — APPROVED** [review]

**Comment by alexander-p2p** [comment]
Let's add C/C++ ?

**Pull request merged**